services/order.py

```diff
@@ -24,7 +25,7 @@ def create_order(tickets: list, username: str, date: str = None) -> Order:
         return new_order
 
 
-def get_orders(username: str = None) -> Order:
+def get_orders(username: str = None) -> QuerySet:
     if username:
         return Order.objects.filter(user__username=username)
-    return Order.objects.all().order_by("-id")
+    return Order.objects.order_by("-id")

```

Comment:
By the way, `order_by()` is not necessary here; it seems that there was a bug that made tests fail on some students' machines, but it should be fixed by now. The tests should pass on GitHub without ordering here even if they fail on your computer.

Reply:
It's ok) Corrected also